src/sentry/web/api.py

```diff
@@ -46,6 +74,11 @@ def robots_txt(request):
     return HttpResponse(ROBOTS_DISALLOW_ALL, content_type="text/plain")
 
 
+@cache_control(max_age=3600, public=True)
+def security_txt(request):
+    return HttpResponse(SECURITY, content_type="text/plain")
```

Comment:
should this be dependent on mode?  I would sorta expect that self-hosted instances would be out of scope for security bounty